pkg/controller/kuberneteseventsource/resources/containersource.go

```diff
@@ -26,14 +26,19 @@ import (
 // MakeContainerSource generates, but does not create, a ContainerSource for the
 // given KubernetesEventSource.
 func MakeContainerSource(source *sourcesv1alpha1.KubernetesEventSource, receiveAdapterImage string) *sourcesv1alpha1.ContainerSource {
+	namespace := source.Spec.Namespace
```

Comment:
My preference is to do this in a webhook, rather than in the controller itself. That way the applied default is easily visible to the user (although in this case, it is simple enough that I think everyone would understand what is happening).